packages/icon/components/Icon.tsx

```diff
@@ -26,7 +24,7 @@ export interface IconProps {
   /**
    * The id of the SVG symbol we're rendering from a generated sprite
    */
-  shape: SystemIcons | ProductIcons;
+  shape: string;
```

Comment:
Hm I don't think we should necessarily change this to be string for all of these.
I will have to think about what to do here further. Would love to hear thoughts from anyone else on @d2iq/ui-team on this.